lib/LMSManagers/LMSNetworkManager.php

```diff
@@ -879,4 +881,86 @@ public function getFirstFreeAddress($netid)
         }
         return $ip;
     }
+
+    public function GetVlanList()
+    {
+        return $this->db->GetAll(
```

Comment:
Może warto od razu użyć GetAllByKey(), żeby zaindeksować automatycznie wynik przez id vlanu?
Może się czasem do fajnych rzeczy przydać.

Reply:
Jeśli ma być bardziej koszerne to ok. Zmienione w ostatnim commicie.